pydantic/main.py

```diff
@@ -938,6 +938,32 @@ def __repr_args__(self) -> 'ReprArgs':
 _is_base_model_class_defined = True
 
 
+@overload
+def create_model(
+    __model_name: str,
+    *,
+    __config__: Type[BaseConfig] = None,
+    __base__: None = None,
+    __module__: str = __name__,
+    __validators__: Dict[str, classmethod] = None,
+    **field_definitions: Any,
+) -> Type['BaseModel']:
+    pass
+
+
+@overload
```

Comment:
Im not an expert by any stretch of the imagination, but isn't this overload the same as the final implementation - is that needed?

Reply:
It's required to have all overloads defined.

Please see more info here [PEP 484](https://www.python.org/dev/peps/pep-0484/#id45)